core/src/main/java/io/grpc/ServerBuilder.java

```diff
@@ -41,6 +41,11 @@
  * @param <T> The concrete type of this builder.
  */
 public abstract class ServerBuilder<T extends ServerBuilder<T>> {
+
+  public static ServerBuilder<? extends ServerBuilder<?>> forAddress(String name, int port) {
```

Comment:
So `name` isn't the same "name" as we will have in the future. In the future we will have `forName(String name)` where name would be ":80" or "localhost:80". So I have no plans to ever make use of the "name" parameter to `forAddress`.

Note that in the future when we have name, then both client and server will have a `forName(String)` factory method.